docs/meshstack.gcp.index.md

```diff
@@ -234,3 +234,28 @@ We recommend that configuration include at least 3 characters of the random para
       "%.15s-%.10s-%.3s"
 }
 ```
+
+## Audit Logs for meshfed-service User 
```

Comment:
n: I'm not 100% sure if this should be a separate page instead. But for now I think that's good enough.